lcls-twincat-motion/Library/Library.plcproj

```diff
@@ -16,9 +16,9 @@
     <LibraryReferences>{9dd7de72-3863-4684-ae1f-9a4b8617466d}</LibraryReferences>
     <CombineIds>false</CombineIds>
     <Company>SLAC</Company>
-    <Released>true</Released>
+    <Released>false</Released>
     <Title>lcls-twincat-motion</Title>
-    <ProjectVersion>0.0.2</ProjectVersion>
+    <ProjectVersion>0.0.0</ProjectVersion>
```

Comment:
Version regressed?

Reply:
0.0.0 is the code for "not a release version". If you install a project with a version number on the PLC programming node it overwrites any existing project with that number.

Reply:
Huh, that's... interesting... and well-worth documenting if we don't already have it somewhere.

Reply:
I agree, I forgot about it when making this repo, which caused a problem, which is why it's bundled into here sloppily. The idea is to set versions to 0.0.0 and use https://github.com/pcdshub/tc_release when we want to apply a tag and set a version number.